### DISCUSSION TITLE
⭐️ Starstruck Exchange ⭐️

### DISCUSSION THREAD

**Discussion by elijahpereira** [Schweinepriester/github-profile-achievements#707] [General]
In a new comment on this thread, add whatever public repo that you would like starred to earn the Starstruck achievement. Star mine and I will do the same.

https://github.com/elijahpereira/elijahpereira.github.io

Note: You can see who's actually starred your repo (e.g. "returned the favor") by appending /stargazers to the end of your repo's URL. For example: https://github.com/scarabaeus/scarabaeus/stargazers

**Comment by elijahpereira** [comment]
Feel free to @ me in your message -- otherwise I check this discussion board about once per day.

**Reply by PARKER-X** [reply]
https://github.com/PARKER-X/BarBar @elijahpereira Done

**Reply by DevElkami** [reply]
> this is mine https://github.com/jaenfigueroa/JaenCraft, I return the number of stars is your repositories with more stars

Great, I gave it a star!

**Reply by shambhaveesrivastava12** [reply]
https://github.com/shambhaveesrivastava12/Data-Structures-and-Algorithms
Hey, help me get stars and I'll help you get yours 😊 @elijahpereira DONE

**Comment by 0x4248** [comment]
Hello I have a project called [Lab Space](https://github.com/awesomelewis2007/lab_space). It has a collection of docker images for quick testing of code.

**Reply by PARKER-X** [reply]
Done bro

**Reply by 0x4248** [reply]
Starred back

**Comment by N0rule** [comment]
@elijahpereira https://github.com/N0rule/emi-discord-bot done

**Reply by elijahpereira** [reply]
> @elijahpereira https://github.com/N0rule/emi-discord-bot done

Started

**Reply by ArielMAJ** [reply]
@N0rule I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by DevElkami** [reply]
> @elijahpereira https://github.com/N0rule/emi-discord-bot done

Great, I gave it a star!

**Comment by elijahpereira** [comment]
> Hello I have a project called [Lab Space](https://github.com/awesomelewis2007/lab_space). It has a collection of docker images for quick testing of code.

Done

**Comment by Veercodeprog** [comment]
You can take part in ongoing [coding problem](https://github.com/Veercodeprog/Data-Structures-implementation/discussions/categories/coding-discussion-share-your-solution-with-best-time-complexity-approach) discussion.

A daily problem will be posted, and the answer with the most efficient time complexity submitted first will be selected as the accepted solution.

**Reply by ArielMAJ** [reply]
Ok

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Comment by xommmax** [comment]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰

**Comment by DevElkami** [comment]
Hi everyone,

[Password Vault](https://github.com/DevElkami/PasswordVault) is a personal password vault with firefox password importer. This vault use Advanced Encryption Standard (AES) encryption (Winform and MAUI).

I would appreciate it if you could check it out and give it a star. Thank you.

**Reply by ArielMAJ** [reply]
@DevElkami I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by DevElkami** [reply]
@ArielMAJ : It's done. Thx.

**Comment by YuZhangWang** [comment]
done
[YuZhangWang](https://github.com/YuZhangWang/YuZhangWang)

**Comment by Denisyasyuchenya** [comment]
done, you should too

**Comment by Rantoryu** [comment]
@elijahpereira
Done, please star my repo: https://github.com/Rantoryu/Personal_Website
I will star back anyone in the comment to this!

**Comment by shambhaveesrivastava12** [comment]
https://github.com/shambhaveesrivastava12/Data-Structures-and-Algorithms
Hey, star mine and I will star yours, do tag me 👍

**Reply by ArielMAJ** [reply]
@shambhaveesrivastava12 I starred you, ty for starring my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.
